app/destination/page.js

```diff
@@ -72,19 +71,30 @@ export const Destinations = () => {
       thumbnail: "/destination/image-titan.png",
     },
   ];
-  const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    if (selectedPlanets.includes(name)) {
-      onAddPlanet(selectedPlanets.filter((planet) => planet !== name));
-    } else {
-      onAddPlanet([...selectedPlanets, name]);
-    }
 
+  const onAddOrRemovePlanet = (index) => {
+    // TASK - React 1 week 2
+    onAddPlanet((prevSelectedPlanets) => {
+      if (prevSelectedPlanets.some((p) => p.name === index.name)) {
+        return prevSelectedPlanets.filter((p) => p.name !== index.name);
+      }
+      return [...prevSelectedPlanets, index];
+    });
     console.log(
-      `You selected the following planet: ${name}, with the index of ${index}`
+      `You selected the following planet: ${index.name}, with the index of ${index.name}`
```

Comment:
This looks like a beautiful message but remember that console log is never for users but rather for developers to debug the code while developing it :)